### PR TITLE
feat(Operators): add tags toString to show displayname of operators

### PR DESCRIPTION
**Pull request by christianalfoni**

### CODE REVIEW

File: packages/cerebral/src/operators/merge.js [hunk 1]
```diff
@@ -17,7 +17,9 @@ export default function (target, ...values) {
     }))
   }
 
-  merge.displayName = 'operator.merge'
+  merge.displayName = `operator.merge(${String(target)}, ${values.map((value) => {
+    return String(value)
```
Comment (edgesoft) [inline packages/cerebral/src/operators/merge.js]: Humm maybe we merge a payload that is rather large from the backend.

File: packages/cerebral/src/operators/set.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default function (target, value) {
     }
   }
 
-  set.displayName = 'operator.set'
+  set.displayName = `operator.set(${String(target)}, ${String(value)})`
```
Comment (edgesoft) [inline packages/cerebral/src/operators/set.js]: Could be a large payload.

File: packages/cerebral/src/operators/merge.js (hunk 1 above)
Comment (christianalfoni) [inline packages/cerebral/src/operators/merge.js]: It is not the value, it is just the string representation of the tag... so basically displays 

```js
"state`foo.bar`" 
```

not the value :)

File: packages/cerebral/src/operators/set.js (hunk 2 above)
Comment (christianalfoni) [inline packages/cerebral/src/operators/set.js]: Same as above ;-)

Comment (edgesoft) [inline packages/cerebral/src/operators/set.js]: Ok, cool my mistake

File: packages/cerebral/src/operators/merge.js (hunk 1 above)
Comment (edgesoft) [inline packages/cerebral/src/operators/merge.js]: Ok, cool my mistake

**Comment by Guria** [comment]
/home/travis/build/cerebral/cerebral/packages/cerebral/src/tags/index.test.js:77:34: Unexpected template string expression.

**Comment by christianalfoni** [comment]
Fixing it now :)

**Pull request merged**